TDP/tdp-20.2.md

```diff
@@ -0,0 +1,38 @@
+---
+tdp: 20.2
+title: MTA/WETH Earn reward changes
+status: Implemented
+author: James Eddington (@jameseddington)
+discussions-to: https://forum.mstable.org/t/mdp20-2-mta-weth-earn-reward-changes/423
+created: 2021-03-29
+---
+
+## Simple Summary
+
+The mStableDAO is seeking to open a discussion around the potential reduction of rewards for the MTA/WETH Uniswap pool in order to reduce the total amount of MTA being emitted, and to focus incentives upon mStable’s new Feeder pool architecture.
+
+## Abstract
+
+As set out in [MCCP 4](../MCCP/mccp-4.md), it has been proposed previously that MTA’s token emission can be optimised. However, whilst MCCP-4 detailed changes to mAssets (mUSD and mBTC) it did not address the rewards going to incentivising the MTA/WETH Uniswap Pool’s 2 liquidity, or Staking V1 rewards. Emissions to these two destinations account for c52% of total outgoing rewards (68,750 of 133,250 MTA each week).
+
+## Motivation
+
+This is being proposed as part of the broader overhaul that was started with MCCP-4, with the intention of comprehensively restructuring MTA reward incentives. This echoes the communicated desire from the Genesis team to lower MTA emission toward liquidity pools outside of the mStable ecosystem, and to refocus rewards where they produce the most utility.
+
+It should be noted too that any loss of liquidity can be partially or fully offset by MTA/DAI liquidity bolstering from the mStableDAO treasury as outlined in [TDP 20.1](tdp-20.1.md) should it be successful.
```

Comment:
please use relative paths like this:
`[TDP 20.1](./tdp-20.1)`

if same directory add the ./ in front
md doesn't need to be added